optimizely/optimizely.py

```diff
@@ -160,7 +160,7 @@ def _validate_user_inputs(self, attributes=None, event_tags=None):
 
         return True
 
-    def _send_impression_event(self, project_config, experiment, variation, user_id, attributes):
+    def _send_impression_event(self, project_config, experiment, variation, user_id, attributes, flag_key, flag_type):
```

Comment:
Missing documentation for new fields. Also, may be good to rethink order of these parameters since it is an internal method.